mora/util.py

```diff
@@ -310,3 +310,36 @@ def log_exception(msg=''):
             data_str,
         )
     )
+
+
+def parse_cpr(
+        number: typing.Union[int, str],
+) -> typing.Tuple[datetime.datetime, int]:
+    if isinstance(number, str):
+        number = int(number)
+
+    rest, code = divmod(number, 10000)
+    rest, year = divmod(rest, 100)
+    rest, month = divmod(rest, 100)
+    rest, day = divmod(rest, 100)
+
+    assert not rest
+
+    # see https://da.wikipedia.org/wiki/CPR-nummer :(
+    if code < 4000:
+        century = 1900
+    elif code < 5000:
+        century = 2000 if year <= 36 else 1900
+    elif code < 9000:
+        century = 2000 if year <= 57 else 1800
+    else:
+        century = 2000 if year <= 36 else 1900
+
+    try:
+        return (
+            datetime.datetime(century + year, month, day,
+                              tzinfo=default_timezone),
+            code,
```

Comment:
Why are we returning the code?

Reply:
> Why are we returning the code?

My idea was that this function simply parsed the CPR number into its two main constituent parts. We're not using it anywhere, though, so I guess _YAGNI_ applies. I'll remove it…